tests/functional/ansibleee_controller_test.go

```
@@ -103,10 +101,7 @@ var _ = Describe("Ansibleee controller", func() {
 					v1alpha1.AnsibleExecutionJobReadyCondition,
 					corev1.ConditionFalse,
 					condition.ErrorReason,
-					fmt.Sprintf(
-						"AnsibleExecutionJob error occured job.name %s job.namespace %s failed",
-						ansibleee.Name, ansibleeeName.Namespace,
-					),
```

it was hiding errors like this:
```
kubectl get osaee   
NAME                       NETWORKATTACHMENTS   STATUS   MESSAGE
ansibleee-playbook-local                        False    AnsibleExecutionJob error occured Job.batch "ansibleee-playbook-local" is invalid: spec.template.spec.containers[0].image: Required value
```